Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,29 @@
+# Release Notes for MultiplayerSample
+
+## O3DE 2305 Release
+
+![2305 Screenshot](Media/2305_screenshot.png)
+
+2305 is the base version of the game. It is expected to be functional and support up to 10 players. 
+
+### Testing Notes
+
+| Platform         | In Editor    | Profile PAKs | Release + PAKs |
+|------------------|--------------|--------------|----------------|
+| Windows          | yes - stable | yes - stable | yes - unstable |
+| Ubuntu           | yes -stable  | no           | no             |
+| Mobile Platforms | n/a          | no           | no             |
+
+MPS was tested using both LAN hosted servers and using AWS EC2 for remote server testing.
+
+### Known Issues
+
+There are some notable known issues:
+
+* On Linux there's a problem with some actor motionsets appearing to have root motion applied: https://github.com/o3de/o3de-multiplayersample/issues/315
+* For the ServerLauncher there are serialization warnings in the log relating to PopcornFX which are harmless, as PopcornFX is ignored on the Server by design.
+* The UX is hardcoded to support 10 players: https://github.com/o3de/o3de-multiplayersample/issues/351. More than 10 players can join the server, but they will not correctly be displayed in the UX.
```

Comment:
Non-UX as well. The server will send player information is arrays that match the “player count” constant